client/my-sites/plan-features/style.scss

```diff
@@ -230,13 +238,23 @@ $plan-features-header-banner-height: 20px;
 	margin: 0;
 	font-size: 14px;
 	color: $gray;
+	position: relative;
 
 	&.is-placeholder {
 		@include placeholder( 23% );
 		height: 50px;
 		margin-top: 5px;
 	}
 }
+.plan-features__footer-desc-longest {
+	visibility: hidden;
```

Comment:
I think we should be ok, since we're not keyword stuffing, but I'll let a few more folks chime in on whether or not this hack is too bad to use. 

Alternatively, we might be able to get away with set height and fading description text.